app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -307,7 +308,7 @@ logger "Starting anaconda <%= @host %> postinstall"
 <%= snippet 'epel' -%>
 <% end -%>
 
-<%= snippet 'redhat_register' if rhel_compatible && os_major < 9 -%>
+<%= snippet 'redhat_register' unless use_rhsm -%>
```

Comment:
This will render `redhat_register` for operating systems that are not RHEL.

Reply:
What OS do you mean? It's intended for Rocky 8 or Alma 8

Reply:
Should I add a guard for it to protect Fedora?  I'm not showing the `rhsm` disabled in https://github.com/rhinstaller/anaconda/blob/master/data/profile.d/fedora.conf so I'd expect it to work there...

Reply:
Shim is right, the `rhsm` command is not available on Fedora: https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#chapter-2-kickstart-commands-in-fedora